Home.md

```diff
@@ -16,22 +16,22 @@ Fork the [wiki repository](https://github.com/LaunchCodeEducation/java-web-devel
-* [Class 2 - Control Flow & Collections](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-2-(ControlFlow-&-Collections))
-* [Class 3 - Classes & Objects](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-3-(Classes-&-Objects))
-* [Class 4 - Classes, Part 2](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-4-(Classes-Part-2))
-* [Class 5 - Unit Testing](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-5-(Unit-Testing))
-* [Class 6 - Inheritance](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-6-(Inheritance))
-* [Class 7 - Interfaces](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-7-(Interfaces))
-* [Class 8 - Intro to Spring and Controllers](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-8-(Intro-to-Spring-and-Controllers))
-* [Class 10 - Thymeleaf Views](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-10-(Thymeleaf-views))
-* [Class 11 - Model Classes and Model Binding](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-11-(Model-Classes-and-Model-Binding))
-* [Class 12 - Model Validation and Enums](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-12-(Model-Validation-and-Enums))
+* [Lesson 1 - Data & Variables](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-1-(Data-&-Variables))
+* [Lesson 2 - Control Flow & Collections](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-2-(ControlFlow-&-Collections))
+* [Lesson 3 - Classes & Objects](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-3-(Classes-&-Objects))
+* [Lesson 4 - Classes, Part 2](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-4-(Classes-Part-2))
+* [Lesson 5 - Unit Testing](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-5-(Unit-Testing))
+* [Lesson 6 - Inheritance](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-6-(Inheritance))
+* [Lesson 7 - Interfaces](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-7-(Interfaces))
+* [Lesson 8 - Intro to Spring and Controllers](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-8-(Intro-to-Spring-and-Controllers))
+* [Lesson 10 - Thymeleaf Views](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-10-(Thymeleaf-views))
+* [Lesson 11 - Model Classes and Model Binding](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-11-(Model-Classes-and-Model-Binding))
+* [Lesson 12 - Model Validation and Enums](https://github.com/LaunchCodeEducation/java-web-development/wiki/Lesson-12-(Model-Validation-and-Enums))
 * [Class 13 - Work Day](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-13-(Work-Day))
```

Comment:
Since we're going by the lesson now, this should be re-formatted to deliver a message about what to do on any work day - since I think most classes will have many of them now too. maybe this is something to go into the more general pages about giving feedback/ how to grade type-pages

Reply:
Done!